pkg/bootstrapper/bootstrapper_test.go

```diff
@@ -1,9 +1,15 @@
 package bootstrapper
 
 import (
-	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"log"
```

Comment:
Why to bring log here?

Reply:
This is due to `log.Fatal()` usage which I will remove.